test/src/com/xilinx/rapidwright/util/TestCodeGenerator.java

```diff
@@ -112,4 +131,65 @@ public void testTestNetGenerator() {
                 + "\n";
         Assertions.assertEquals(expectedString, actualString);
     }
+
+    @Test
+    public void testGenCodeForTestSite(@TempDir Path tempDir) {
+        Design d = RapidWrightDCP.loadDCP("microblazeAndILA_3pblocks.dcp");
+        SiteInst si = d.getSiteInstFromSiteName("SLICE_X36Y97");
+
+        Path javaFile = tempDir.resolve(CodeGenerator.TEST_SITE_INST_CLASS_NAME + ".java");
+        try (PrintStream ps = new PrintStream(javaFile.toString())) {
+            CodeGenerator.genCodeForTestSite(si, ps, false);
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
```

Comment:
```suggestion
            throw new RuntimeException(e);
```

Or just let it propagate out and get caught by Junit.